arc_solvers/models/qa/multi_choice/qa_multi_choice_max_att.py

```diff
@@ -0,0 +1,247 @@
+
+        if question_output_dim != choice_output_dim:
+            raise ConfigurationError("Output dimension of the question_encoder (dim: {}) "
+                                     "and choice_encoder (dim: {})"
+                                     "must match! "
+                                     .format(question_output_dim,
+                                             choice_output_dim))
+
+        # Check input tensor dimensions for the question to choices attention (similarity function)
+        if hasattr(att_question_to_choice, "tensor_1_dim"):
+            tensor_1_dim = att_question_to_choice.tensor_1_dim
+            if tensor_1_dim != question_output_dim:
+                raise ConfigurationError("Output dimension of the question_encoder (dim: {}) "
+                                         "and tensor_1_dim (dim: {}) of att_question_to_choice"
+                                         "must match! "
+                                         .format(question_output_dim,
+                                                 tensor_1_dim))
+
+        if hasattr(att_question_to_choice, "tensor_2_dim"):
+            tensor_2_dim = att_question_to_choice.tensor_2_dim
+            if tensor_2_dim != question_output_dim:
+                raise ConfigurationError("Output dimension of the choice_encoder (dim: {}) "
+                                         "and tensor_2_dim (dim: {}) of att_question_to_choice"
+                                         "must match! "
+                                         .format(choice_output_dim,
+                                                 tensor_2_dim))
+
+        self._matrix_attention_question_to_choice = MatrixAttention(att_question_to_choice)
+
+        self._accuracy = CategoricalAccuracy()
+        self._loss = torch.nn.CrossEntropyLoss()
+
+        initializer(self)
+
+    def forward(self,  # type: ignore
+                question: Dict[str, torch.LongTensor],
+                choices_list: Dict[str, torch.LongTensor],
+                label: torch.IntTensor = None,
+                metadata: List[Dict[str, Any]] = None,
+                ) -> Dict[str, torch.Tensor]:
+        # pylint: disable=arguments-differ
+        """
+        Parameters
+        ----------
+        question : Dict[str, torch.LongTensor]
+            From a ``TextField``
+        choices_list : Dict[str, torch.LongTensor]
+            From a ``List[TextField]``
+        label : torch.IntTensor, optional (default = None)
+            From a ``LabelField``
+
+        Returns
+        -------
+        An output dictionary consisting of:
+
+        label_logits : torch.FloatTensor
+            A tensor of shape ``(batch_size, num_labels)`` representing unnormalised log
+            probabilities of the entailment label.
+        label_probs : torch.FloatTensor
+            A tensor of shape ``(batch_size, num_labels)`` representing probabilities of the
+            entailment label.
```

Comment:
"entailment"? Is this a copy and paste error, or do you think of it as "Does the question stem entail the answer choice?"

Reply:
This was a copy-paste error. Fixed it.